hazelcast/src/main/java/com/hazelcast/map/impl/BinaryMapEntryCostEstimator.java

```diff
@@ -50,6 +51,9 @@ public long getEstimate() {
         return estimate;
     }
 
+    @SuppressFBWarnings(value = "VO_VOLATILE_INCREMENT",
+            justification = "We have the guarantee that only a single partition thread at any given time can change the volatile"
```

Comment:
Since this is accessed from a single partition thread, why don't we remove volatile instead? It looks useless to me.

Reply:
@mustafaiman: it's mutated by a single thread, but read by multiple threads. I guess it's a good idea to add it to the justification.

Reply:
Right, I assumed the estimate was aggregated somehow using partition operations rather than directly reading record stores. Disregard my comment.

Reply:
I added the info about reads to the justification.